server/views/outcomes/casesToResult.njk

```diff
@@ -120,7 +120,7 @@
     {% if params.filtersApplied %}
       <span class="govuk-heading-m">0 cases</span>
```

Comment:
Same here, is this expected? captured somewhere else?

Reply:
Ticket has been updated!